Tests/StackClusterTests.swift

```diff
@@ -112,7 +112,7 @@ class StackClusterTests: QuickSpec {
                     labelComponent(title: "1", color: .red, labelSize: size),
                     labelComponent(title: "2", color: .red, labelSize: size),
                     labelComponent(title: "3", color: .red, labelSize: size),
-                    labelComponent(title: "4", color: .red, labelSize: size),
+                    labelComponent(title: "4", color: .red, labelSize: size)
```

Comment:
I like trailing comma in collections because you can easily add or move lines... I know swiftlint complains with recent versions but we might disable the rule eventually